website/content/docs/concepts/iam.mdx

```diff
@@ -0,0 +1,103 @@
+[permissions](https://developer.hashicorp.com/boundary/docs/concepts/security/permissions) which are granted to any principal (user or group) from any scope. Roles 
+can be contained by any scope, and can specify which scope the permissions will apply to.
+
+## Access Management
+#### Configuring Users
+When adding new users to Boundary using the password auth method, there are 3 steps to follow. Note: for OIDC auth methods, the following will be auto-generated using 
+information provided by the identity provider.
+1. **Auth Method** - Choose the scope and the auth method that the user will use to authenticate to Boundary. If the desired auth method does not exist, a new one needs 
+to be created.
+2. **Account** - After the auth method is selected, an account can be created under the auth method.
+3. **User** - In the same scope, create a new user, and then attach the new account to the new user. The user is now able to authenticate to Boundary.
+
+#### Granting Permissions
+When setting up access controls for a user, it is important to first consider which scopes the user needs access to. **Each scope needs its own role** which the user must 
+be added to. After adding the user to each role, grants need to be written to give the user permissions to perform actions in each role. 
+
+## Examples 
+Here are a few examples of access management for personas, using this sample scope structure. In this structure, there are 7 scopes overall: 1 global, 2 Orgs, and 4 
+Projects. To learn more about writing grants, see here.
+
+![IAM Example](/img/iam-example-structure.png)
+
+#### Boundary Admin
+1. All 7 scopes **each** require a role (ex. “admin”)
+2. Each role needs to have the user added to it
+3. Each role needs to have the grants of
+
+    `id=*;type=*;actions=*`
+
+#### Boundary Viewer
+1. All 7 scopes **each** require a role (ex. “viewer”)
+2. Each role needs to have the user added to it
+3. Each role needs to have the grants of
+
+    `id=*;type=*;actions=read,list`
+
+#### Org_A Admin
+1. The following scopes **each** require a role (ex. “admin”)
+    - Org_A
+    - Project_1
+    - Project_2
+2. Each role needs to have the user added to it
+3. Each role needs to have the grants of
+
+    `id=*;type=*;actions=*`
+
+#### Project_1 Admin
+1. Project_1 requires a role (ex. “admin”)
+2. The role needs to have the user added to it
+3. The role needs to have the grants of
+
+    `id=*;type=*;actions=*`
+
+#### Project_1 Target Access
+1. Project_1 requires a role (ex. “target_access”)
+2. The role needs to have the user added to it
+3. The role needs to have the grants of
+
+    `id=*;type=target;actions=list,read,authorize-session`
+    `id=*;type=session;actions=read:self,cancel:self,list`
+
+-> **Tip:** The same roles can be used across “Boundary Admin”, “Org_A Admin”, and “Project_1 Admin” since the scopes and grants are the same. The only difference is 
+the users that are added to each role. 
```

Comment:
```suggestion
-> **Tip:** You can use the same roles across “Boundary Admin”, “Org_A Admin”, and “Project_1 Admin” since the scopes and grants are the same. The only difference is 
the users that are added to each role. 
```